build/common/installer/conf/azm-containers-parser-multiline.conf

```diff
@@ -0,0 +1,17 @@
+# [MULTILINE_PARSER]
+#     name          dotnet-regex
+#     type          regex
+#     flush_timeout 1000
```

Comment:
what's the units of this value? is it 1000 seconds? how this being determined? should we make this configurable through configmap?

Reply:
yes it's in seconds. I don't think we need to expose this for private preview

Reply:
@pfrcks , what's the rationale behind picking 1000 seconds?  since flush timeout is 1000 seconds, so the stitched logs shows after 1000 seconds in LA? and also this might cause data loss in high log volume environment (with back pressure) due to accumulation of the logs for this duration , right?